src/components/popover-container/popover-container.test.js

```diff
@@ -358,4 +316,62 @@ context("Test for Popover Container component", () => {
       }
     );
   });
+
+  describe("Accessibility tests for Popover Container component", () => {
+    it("should pass accessibilty tests for Popover Container Default story", () => {
+      CypressMountWithProviders(<stories.Default />);
+
+      popoverSettingsIcon().click();
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibilty tests for Popover Container Title story", () => {
+      CypressMountWithProviders(<stories.Title />);
+
+      popoverSettingsIcon().click();
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibilty tests for Popover Container Position story", () => {
+      CypressMountWithProviders(<stories.Position />);
+
+      popoverSettingsIcon().click();
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibilty tests for Popover Container CoverButton story", () => {
+      CypressMountWithProviders(<stories.CoverButton />);
+
+      popoverSettingsIcon().click();
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibilty tests for Popover Container RenderProps story", () => {
+      CypressMountWithProviders(<stories.RenderProps />);
+
+      popoverSettingsIcon().click();
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibilty tests for Popover Container Controlled story", () => {
+      CypressMountWithProviders(<stories.Controlled />);
+
+      popoverSettingsIcon().click();
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibilty tests for Popover Container Complex story", () => {
+      CypressMountWithProviders(<stories.Complex />);
+
+      popoverSettingsIcon().click();
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibilty tests for Popover Container Filter story", () => {
```

Comment:
For the filter story, I think we also need to check the accessibility after the button is clicked